todo_app/templates/index.html

```diff
@@ -10,9 +10,34 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+        {% for item in items %}
+          <div class="card" style="width: 18rem;">
+            <img src="..." class="card-img-top" alt="...">
+            <div class="card-body">
+              <h5 class="card-title">[{{ item.id }}] {{ item.title }}</h5>
+              <p class="card-text">{{ item.status }}</p>
+            </div>
+            <div class="card-body">
+              {% if item.status != "Done" %}
+                <a class="card-link" href="{{ url_for('mark_as_done', itemId=item.id) }}">Done</a>              
+              {% endif %}
+              <a class="card-link" href="{{ url_for('delete', itemId=item.id) }}">Delete</a> 
+            </div>
+          </div>
+        {% endfor %}
+    </div>
+  </div>  
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Add New Item</h2>
+      <form action="/items" method="POST" class="form-group">
+        <div class="form-group">
+          <input type="text" class="form-control" name="title" id="title" aria-describedby="titleHelp" placeholder="Title">
```

Comment:
There's no element with id `titleHelp` element so the `aria-describedby` will not do anything here.

Reply:
thanks, removed